### DISCUSSION TITLE
Http 404 for app.planetscale.com and access denied when I try to login

### DISCUSSION THREAD

**Discussion by DuckBurnIncense** [planetscale/discussion#250] [Bug Reports] [answered]
![图像 001](https://user-images.githubusercontent.com/87637270/184516483-9bba389f-4e79-4480-81a7-aba5f286e863.png)

When I click the link in the email to wake up my database it show me `This app.planetscale.com page can’t be found`

![图像 002](https://user-images.githubusercontent.com/87637270/184516552-7aea5765-191b-458e-8197-6d52c3294767.png)

And when I try to login 

![图像 003](https://user-images.githubusercontent.com/87637270/184516513-2d4b08c7-67ff-45d0-9437-65d47a93e64b.png)

it show me

```json
{
    "error": "access denied"
}
```

![图像 004](https://user-images.githubusercontent.com/87637270/184516524-fcf30211-fc25-4cff-9f5a-24333360fc2b.png)

I have tried:

- change brower;
- clean cookies
- use InPrivate window

<small>I’m not a native English speaker, so please excuse my grammar mistakes. (I'm a Chinese)</small>

**Comment by maple-yf** [comment]
Same problem, and Chinese too.

**Reply by flower4486** [reply]
墙了吧，翻出来可以啊

**Reply by DuckBurnIncense** [reply]
> 墙了吧，翻出来可以啊

好像确实是的, 但为啥planetscale要屏蔽中国用户呢

**Reply by maple-yf** [reply]
我翻出来也不行。。。还是access denied

**Reply by DuckBurnIncense** [reply]
> 我翻出来也不行。。。还是access denied

刚试了一下, 用VPN还是可以的哦

**Reply by maple-yf** [reply]
确实得用VPN

**Reply by flower4486** [reply]
电脑的话，如果你用的是v2ray你要注意他有几个不同的模式，我们一般用的是pac代理，他不会改变你的ip地址，所以你的ip还是在大陆，会被planescale拒绝，你要用http代理模式。你可以在百度搜索ip来确认自己当前的IP地址。

**Comment by johnyerhot** [comment, marked as answer]
Hello folks – at this time PlanetScale is no longer available inside of the People's Republic of China. That is the reason for the error messages being discussed here.

If you are outside the People's Republic of China and receiving that error please let us know at `support@planetscale.com`.

We know it's not an ideal situation and appreciate your understanding.

**Comment by flower4486** [comment]
电脑的话，如果你用的是v2ray你要注意他有几个不同的模式，我们一般用的是pac代理，他不会改变你的ip地址，所以你的ip还是在大陆，会被planescale拒绝，你要用http代理模式。你可以在百度搜索ip来确认自己当前的IP地址。

**Reply by DuckBurnIncense** [reply]
谢谢提醒, 不过我用的是`Clash`

**Reply by flower4486** [reply]
> 谢谢提醒, 不过我用的是`Clash`

没事，不过理论应该差不多，不只我们在墙人家，人家也在屏蔽我们哈哈

**Reply by maple-yf** [reply]
我也用的clashx，直接手动把planetscale.com配置到规则里了，哈哈哈哈。
